tests/client-server.py

```diff
@@ -22,30 +22,34 @@ def port_is_open(host, port):
 
 
 def wait_tcp_port(host, port):
-    for _ in range(MAX_TRIES):
+    for i in range(MAX_TRIES):
         if port_is_open(host, port):
             break
         else:
-            print("{} - still trying to connect to {}:{}"
-                  .format(time.strftime("%c"), host, port))
-            time.sleep(0.5)
+            if i > 0:
+                print("client-server.py: {} - still trying to connect to {}:{}"
```

Comment:
nit: I'd probably use an f-string here:
```suggestion
                print(f"client-server.py: {time.strftime('%c')} - still trying to connect to {host}:{port}")
```

Reply:
Good call, done.